richie/settings.py

```diff
@@ -234,6 +272,49 @@ class Base(DRFMixin, ElasticSearchMixin, Configuration):
         "easy_thumbnails.processors.background",
     )
 
+    # Sentry
+    #
+    # Nota bene: the DJANGO_SENTRY_DSN environment variable should be defined to
+    # report failures to Sentry
+    RAVEN_CONFIG = {
+        "dsn": values.Value(environ_name="SENTRY_DSN"),
+        "release": get_release(),
+        "environment": values.Value(environ_name="CONFIGURATION"),
+    }
+
+    LOGGING = {
+        "version": 1,
+        "disable_existing_loggers": True,
+        "root": {"level": "WARNING", "handlers": ["sentry"]},
+        "formatters": {
+            "verbose": {
+                "format": "%(levelname)s %(asctime)s %(module)s "
+                "%(process)d %(thread)d %(message)s"
+            }
+        },
+        "handlers": {
+            "sentry": {
+                "level": "ERROR",  # To capture more than ERROR, change to WARNING, INFO, etc.
```

Comment:
Don't we actually want to be able to send more than just errors to sentry?

I'm thinking events like ES index automated management, synchronization with fun-apps/openedx etc. should be able to send INFO and WARNING level events so we have a history and can quickly check how the latest actions went when we need to debug something.

Reply:
`WARNING` seems relevant to me, but not `INFO`: at this level, logs should be sent to syslog and further analyzed/mined with an ELK based dashboard.

Reply:
Ideally yes, but as a first approximation, until we have an ELK instance running, Sentry could be helpful there.

I've seen @sampaccoud do that before, and I think it's better than dropping that data entirely.

Reply:
Maybe add an issue so we remember to take care of this later :)

Reply:
Done in #114.